mesos/deploy.tf

```diff
@@ -0,0 +1,26 @@
+# Generate a cert for each mesos machine and push it into the instances
```

Comment:
I'm being abstraction man here again but we are duplicating same code over and over across the different modules.
We should probably create a generic module and then reuse it as a nested module across the specific modules (docker, kube, etc) with the opinionated parameters